### ISSUE TITLE
port k8s 0.7.3 changes from multiverse PR

### ISSUE THREAD

**Issue by jdef** [mesosphere/universe#348]
xref https://github.com/mesosphere/multiverse/pull/78

**Comment by dellintosh** [comment]
Any updates on this?  Just got our mesosphere cluster set up and was sad to see k8s not in the universe anymore... 😢   I'd love to see it back in there soon!  👍

**Comment by jdef** [comment]
no updates for the moment. we've been focused on other, more immediate
customer priorities. in the meantime there have been some changes to dc/os
that are incompatible with k8s-mesos. so we removed the package rather than
leave it there, broken.

are you interested in running k8s on dc/os in production?

**Comment by dellintosh** [comment]
Yes, we'd be implementing this in production as well as testing environments.  We have been using Kubernetes in our workflow to deploy applications (actually using [Deis workflow](https://deis.com/))... but we'd be open to alternatives if we can get the same `git push` build/deploy process in other implementations...

I recently came across Mesosphere (and DC/OS) and after installing it on a test cluster environment I think it'd make our operations and cluster management easier.  It'd be nice to get K8S running on that so we wouldn't have to change our dev workflow... but again, the main draw is that heroku-like `git push` model that they made so famous...

I'd be open for other suggestions...
